pkg/sdk/testint/secrets_gen_integration_test.go

```diff
@@ -0,0 +1,673 @@
+package testint
+
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers/random"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const Day = 24 * time.Hour
+
+func TestInt_Secrets(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	integrationId := testClientHelper().Ids.RandomAccountObjectIdentifier()
+
+	refreshTokenExpiryTime := time.Now().Add(Day).Format(time.DateOnly)
```

Comment:
Did you try other formats? How Snowflake behaves when we give it a different one? You can check manually.

Reply:
Did you check? If you did it manually, comment above that the format has to be used otherwise, e.g. Snowflake will return error.